src/FSharp.Core/async.fs

```diff
@@ -868,11 +868,7 @@ module AsyncPrimitives =
 
     ///   - Initial cancellation check
     ///   - Call syncCtxt.Post with exception protection. This may fail as it is arbitrary user code
-#if BUILDING_WITH_LKG || NO_NULLCHECKING_LIB_SUPPORT
     let CreateSwitchToAsync (syncCtxt: SynchronizationContext) =
```

Comment:
shall fsi and fs be synchromised?

Reply:
This method is not part of the public signature. its in async.fs only.